modules/veteran/spec/models/veteran/service/representative_spec.rb

```diff
@@ -73,14 +73,15 @@ def basic_attributes
       it 'can find a user with a suffix' do
         expect(Veteran::Service::Representative.all_for_user(
           first_name: identity.first_name,
-          last_name: "#{identity.last_name} III",
+          last_name: identity.last_name,
+          suffix: 'Jr',
           poa_code: 'A1Q'
         ).first.poa_codes).to include('A1Q')
       end
```

Comment:
Should we create a test user with a suffix in their last name, so we can exercise the case where we find a user by their suffix? This test will strip the suffix (`Jr`) because there is no `lincoln jr` in the database, which conceals the error in our return logic.